stats.go

```diff
@@ -285,12 +288,20 @@ type timespan struct {
 	start time.Time
 }
 
+// Complete records the elapsed time in nanos, and returns the elapsed duration.
 func (ts *timespan) Complete() time.Duration {
 	d := time.Now().Sub(ts.start)
 	ts.timer.time(d)
 	return d
 }
 
+// CompleteAsMS records the elapsed time in ms, and returns the elapsed duration.
+func (ts *timespan) CompleteAsMS() time.Duration {
+	d := time.Now().Sub(ts.start)
+	ts.timer.time(d / time.Millisecond)
```

Comment:
`timer.time()` is:
```golang
func (t *timer) time(dur time.Duration) {
	t.sink.FlushTimer(t.name, float64(dur/time.Microsecond))
}
```

So you need to multiply by 1000 or `time.Millisecond / time.Microsecond` not divide.  This currently gives you nanoseconds not milliseconds.

Reply:
doesn't this currently give micros / 10e6 then, so seconds as written?

Reply:
Yep, this gives seconds.  But the method name and comment indicate that this records milliseconds.